shared/src/jvmMain/kotlin/dev/johnoreilly/confetti/di/KoinJVM.kt

```diff
@@ -29,4 +29,4 @@ actual fun platformModule() = module {
     }
 }
 
-actual fun getDatabaseName(conference: String) = "jdbc:sqlite:$conference.db"
+actual fun getDatabaseName(conference: String, uid: String?) = "jdbc:sqlite:$conference$uid.db"
```

Comment:
This is surprising, do we need multiple concurrent users?

Reply:
It's more to avoid cloberring the bookmarks cache between users. If we don't do this, we have to keep track of the uid and remove everything on logout to make sure we don't load wrong data if someone else logs in.